android/app/build.gradle

```diff
@@ -181,12 +183,13 @@ android {
             applicationId "com.espeocreatereactnativeapp.dev"
             dimension "default"
         }
-         stage {
+        stage {
             applicationId "com.espeocreatereactnativeapp.stage"
             dimension "default"
         }
         prod {
-            applicationId "com.espeocreatereactnativeapp.prod"
+            // applicationId "com.espeocreatereactnativeapp.prodd" Fastlane deploy
```

Comment:
leave 'prod' as default, and add in comment that prodd -> is for your local play store tests using fastlane.